iroh-resolver/src/unixfs_builder.rs

```diff
@@ -168,6 +170,8 @@ pub struct File {
     chunker: Chunker,
 }
 
+unsafe impl Send for File {}
```

Comment:
Not thrilled about unsafe code. @dignifiedquire what's our general policy on that front?

Reply:
No, this is not safe to do. Each piece of the struct needs to be made `Send`